src/main/resources/application.conf

```diff
@@ -38,110 +46,221 @@ etc-client {
+
+    # Difficulty bomb pause block number
+    # Doc: https://github.com/ethereumproject/ECIPs/blob/master/ECIPs/ECIP-1010.md
     difficulty-bomb-pause-block-number = "3000000"
+
+    # Difficulty bomb continuation block number
+    # Doc: https://github.com/ethereumproject/ECIPs/blob/master/ECIPs/ECIP-1010.md
     difficulty-bomb-continue-block-number = "5000000"
 
-    // Doc: https://blog.ethereum.org/2016/07/20/hard-fork-completed/
+    # DAO fork block number (Ethereum HF/Classic split)
+    # https://blog.ethereum.org/2016/07/20/hard-fork-completed/
     dao-fork-block-number = "1920000"
-    dao-fork-block-total-difficulty = "39490964433395682584"
+
+    # The hash of the accepted DAO fork block
     dao-fork-block-hash = "94365e3a8c0b35089c1d1195081fe7489b528a84b22199c916180db8b28ade7f"
 
+    # Starting nonce a an empty account. Some networks (like Morden) use different values.
     account-start-nonce = "0"
 
+    # The ID of the accepted chain
     chain-id = "3d"
 
+    # Custom genesis JSON file
     custom-genesis-file = null
 
-    # ECIP-1017
+    # Monetary policy parameters
+    # Doc: https://github.com/ethereumproject/ECIPs/blob/master/ECIPs/ECIP-1017.md
     monetary-policy {
+      # Block reward in the first era
       first-era-block-reward = "5000000000000000000"
+
+      # Monetary policy era duration in number of blocks
       era-duration = 5000000
+
+      # Rate at which rewards get reduced in successive eras.
+      # Value in range [0.0, 1.0]
       reward-reduction-rate = 0.2
     }
   }
 
-  fast-sync {
+  sync {
+    # Whether to enable fast-sync
     do-fast-sync = true
+
+    # TODO: explain
     peers-scan-interval = 3.seconds
+
+    # Duration for blacklisting a peer
+    # TODO: explain better, reasons for blacklisting?
     blacklist-duration = 30.seconds
+
+    # TODO: explain
     start-retry-interval = 5.seconds
+
+    # TODO: explain
     sync-retry-interval = 5.seconds
+
+    # TODO: explain
     peer-response-timeout = 10.seconds
+
+    # TODO: explain
     print-status-interval = 2.seconds
+
+    # TODO: explain
     persist-state-snapshot-interval = 1.minute
 
+    # Maximum concurrent requests when in fast-sync mode
     max-concurrent-requests = 50
+
+    # Requested number of block headers when syncing from other peers
     block-headers-per-request = 2048
+
+    # Requested number of block bodies when syncing from other peers
     block-bodies-per-request = 128
+
+    # Requested number of TX receipts when syncing from other peers
     receipts-per-request = 60
+
+    # Requested number of MPT nodes when syncing from other peers
     nodes-per-request = 1000
+
+    # Minimum number of peers required to determine the target block for fast-sync
     min-peers-to-choose-target-block = 2
+
+    # During fast-sync when most up to date block is determined from peers, the actual target block number
+    # will be decreased by this value
     target-block-offset = 500
 
+    # TODO: explain
```

Comment:
this is used as interval to schedule consecutive request to peers when we reach top of the chain and peer does not have any newer blocks
we schedule request to check if peer has new block after 1 second